tests/ZonedDateTimeTest.php

```diff
@@ -814,24 +814,38 @@ public function providerToNativeDateTime(): array
         ];
     }
 
-    public function testJsonSerialize(): void
+    /** @dataProvider providerToString */
+    public function testJsonSerialize(string $localDateTime, string $timeZoneIso, string $expectedString): void
```

Comment:
I'm not sure if the time zone database is governed by ISO, so this may be misleading. Just `string $timeZone`?